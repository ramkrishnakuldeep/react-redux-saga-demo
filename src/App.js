import React, { Component } from 'react';
import { Route, Switch, withRouter } from 'react-router-dom';
import ItemListContainer from './containers/ItemListContainer';
import ItemDetailContainer from './containers/ItemDetailContainer';

class App extends Component {
    render () {
        return (
            <React.Fragment>
                <Switch>
                    <Route exact path="/" component={ItemListContainer} />
                    <Route path={'/items/:itemId'} component={ItemDetailContainer}/>
                </Switch>
            </React.Fragment>
        );
    }
}

export default withRouter(App);
