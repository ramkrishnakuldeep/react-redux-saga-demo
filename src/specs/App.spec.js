import React from 'react';
import { shallow } from '../enzyme';
import App from '../App';

describe('App component', () => {
    it('initial render', () => {
        const component = shallow(<App />);
        expect(component).toMatchSnapshot();
    });
});
