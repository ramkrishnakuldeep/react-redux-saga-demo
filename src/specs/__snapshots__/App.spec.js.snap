// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component initial render 1`] = `
<Route>
  <Component />
</Route>
`;
